DOCS-2364 Alerting new documentation structure (#11576)
DataDog/documentation: content/en/monitors/create/monitor_configuration.md

| Option                                   | Description                    |
|------------------------------------------|--------------------------------|
| Alert&nbsp;threshold&nbsp;**(required)** | The value used to trigger an alert notification. |
| Warning&nbsp;threshold                   | The value used to trigger a warning notification. |
| Alert&nbsp;recovery&nbsp;threshold       | An optional threshold to indicate an additional condition for alert recovery. |
| Warning&nbsp;recovery&nbsp;threshold     | An optional threshold to indicate an additional condition for warning recovery. |

As you change a threshold, the preview graph in the editor displays a marker showing the cutoff point.

{{< img src="/monitors/create/preview_graph_thresholds.png" alt="Thresholds preview graph" style="width:100%;">}}

**Note**: When entering decimal values for thresholds, if your value is `<1`, add a leading `0` to the number. For example, use `0.5`, not `.5`.

[1]: /monitors/guide/as-count-in-monitor-evaluations/
[2]: /monitors/faq/what-are-recovery-thresholds/
{{% /tab %}}

> {{% tab "Check Alert" %}}

Reviewer: ```suggestion
{{% tab "Check alert" %}}
```

> A check alert tracks consecutive statuses submitted per check grouping and compares it to your thresholds. Set up the check alert to :

Reviewer: ```suggestion
A check alert tracks consecutive statuses submitted per check grouping and compares it to your thresholds. Set up the check alert to:
```

1. Trigger the alert after selected consecutive failures: `<NUMBER>`

    Each check run submits a single status of `OK`, `WARN`, or `CRITICAL`. Choose how many consecutive runs with the `WARN` and `CRITICAL` status trigger a notification. For example, your process might have a single blip where connection fails. If you set this value to `> 1`, the blip is ignored but a problem with more than one consecutive failure triggers a notification.

    {{< img src="/monitors/create/check_thresholds_alert_warn.png" alt="Check thresholds Alert/Warn" style="width:90%;">}}

2. Resolve the alert after selected consecutive successes: `<NUMBER>`

    Choose how many consecutive runs with the `OK` status resolves the alert.

    {{< img src="/monitors/create/check_thresholds_recovery.png" alt="Check thresholds Recovery" style="width:90%;">}}

> See the [process check][1], [integration check][2] and [custom check][3] monitors documentation for more information on how to configure check alerts.

Reviewer: ```suggestion
See the documentation for [process check][1], [integration check][2], and [custom check][3] monitors for more information on configuring check alerts.
```

Auto-resolve works when data is no longer being submitted. Monitors do not auto-resolve from an ALERT or WARN state. If data is still being submitted, the [renotify][6] feature can be utilized to let your team know when an issue is not resolved.

For some metrics that report periodically, it may make sense for triggered alerts to auto-resolve after a certain time period. For example, if you have a counter that reports only when an error is logged, the alert never resolves because the metric never reports `0` as the number of errors. In this case, set your alert to resolve after a certain time of inactivity on the metric. **Note**: If a monitor auto-resolves and the value of the query does not meet the recovery threshold at the next evaluation, the monitor triggers an alert again.

In most cases this setting is not useful because you only want an alert to resolve once it is actually fixed. So, in general, it makes sense to leave this as `[Never]` so alerts only resolve when the metric is above or below the set threshold.

#### New group delay

Delay the evaluation start by `N` seconds for new groups.

> The time (in seconds) to wait before starting alerting, to allow newly created groups to boot and applications to fully start. This should be a non-negative integer.

Reviewer: ```suggestion
The time (in seconds) to wait before starting alerts to allow newly created groups to boot and applications to fully start. This should be a non-negative integer.
```
